src/jenkins_job_insight/cli/main.py

```
@@ -1805,6 +1808,134 @@ def admin_users_change_role(
 
+# -- Metadata -----------------------------------------------------------------
+
+
+@metadata_app.command("list")
+def metadata_list(
+    team: str = typer.Option("", "--team", help="Filter by team."),
+    tier: str = typer.Option("", "--tier", help="Filter by tier."),
+    version: str = typer.Option("", "--version", help="Filter by version."),
+    label: list[str] = typer.Option(
+        [], "--label", "-l", help="Filter by label (can repeat)."
+    ),
```

Ruff (0.15.10)

[warning] Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)